### DISCUSSION TITLE
Inputmask better implementation

### DISCUSSION THREAD

**Discussion by DcassianoMoveedu** [breezefront/community#327] [1. Help] [answered]
Hi Guys,

I need a help about implements InputMask to input fields without use jquery plugins for that. 
Someone implements this using only breeeze resources?

**Comment by vovayatsyuk** [comment, marked as answer]
I created a breeze-examples repo and added the first example with vanillaTextMask usage.

Feel free to reuse and change its code as you need: https://github.com/breezefront/module-breeze-examples

**Reply by DcassianoMoveedu** [reply]
@vovayatsyuk works fine! Thanks you for fast reply! 
Now i can set masks in brazilian formats!
